ComputerVision/Rest/ExtractText/Program.cs

```diff
@@ -0,0 +1,217 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace CSHttpClientSample
+{
+    static class ExtractText
+    {
+        // Replace with your valid subscription key.
+        const string subscriptionKey = "0123456789abcdef0123456789ABCDEF";
+
+        // You must use the same Azure region in your REST API method as you used to get your subscription keys. Free trial subscription keys are generated in the "westus" region.
+        const string uriBase = "https://westus.api.cognitive.microsoft.com/vision/v2.0/read/core/asyncBatchAnalyze";
+
+        static void Main()
+        {
+            Console.WriteLine("Extract text in images:");
+
+            string imageFilePath = @"Images\sample2.png";
+            string remoteImageUrl = "https://github.com/harishkrishnav/cognitive-services-dotnet-sdk-samples/raw/master/ComputerVision/Images/sample0.png";
+
+            if (File.Exists(imageFilePath))
+            {
+                // Call the REST API method.
+                Console.WriteLine("\nWait a moment for the results to appear.\n");
+                ExtractTextFromStreamAsync(imageFilePath).Wait();
+            }
+            else
+            {
+                Console.WriteLine("\nInvalid file path");
+            }
+
+            if (Uri.IsWellFormedUriString(remoteImageUrl, UriKind.Absolute))
+            {
+                ExtractTextFromUrlAsync(remoteImageUrl).Wait();
+            }
+            else
+            {
+                Console.WriteLine("\nInvalid remote image url:\n{0} \n", remoteImageUrl);
+            }
+
+            Console.WriteLine("\nPress Enter to exit...");
+            Console.ReadLine();
+        }
+
+        /// <summary>
+        /// Gets the text from the specified image file by using
+        /// the Computer Vision REST API.
+        /// </summary>
+        /// <param name="imageFilePath">The image file with text.</param>
+        static async Task ExtractTextFromStreamAsync(string imageFilePath)
+        {
+            try
+            {
+                HttpClient client = new HttpClient();
+
+                // Request headers.
+                client.DefaultRequestHeaders.Add("Ocp-Apim-Subscription-Key", subscriptionKey);
+
+                // Request parameter.
+                string requestParameters = "mode=Handwritten";  // replace mode=Printed
```

Comment:
I would make this comment more specific, and promote it to its own line right above. Something like:
// replace mode=Printed if passing in images with printed text
string requestParameters = "mode=Printed";